components/lib/screens/register.dart

```diff
@@ -1,106 +1,199 @@
 import 'package:flutter/material.dart';
 import 'package:flutter/services.dart';
+import 'package:travelsafe/authentication.dart';
 
 class RegisterPage extends StatefulWidget {
```

Comment:
It could be better if we extract this class to separate file.

Reply:
Which class? I didn't understand

Reply:
I am referring to  RegisterPage  and _RegisterPageState class

Reply:
Sorry for being so late. You are telling me to make `_RegisterPageState`  class in a different file. Is it right

Reply:
Yes, exactly. let me know if you are facing any trouble

Reply:
was this done?

Reply:
@pravinyo I am not getting, why should I do that

Reply:
There is no issue in the code, I suggested this change based on commit history and responsibility of class. One class is handling UI rendering while other is non-ui related. Changes to file causes compilation of both class, I am just making sure this code is easily scalable in future with new feature addition.  @tanmayrauth @ankitshaw  let me know if this can be skipped.